airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -203,23 +207,39 @@ def process(self) -> List["StreamProcessor"]:
             return []
 
         column_names = self.extract_column_names()
+        column_count = len(column_names)
+
+        if column_count == 0:
```

Comment:
is this the logic that lets us ignore the 'raw' Airbyte table?

Reply:
yes, the compilation error there was because there are no columns left to normalize after removing columns prefixed with `_airbyte` so in those cases, we don't produce any normalization models for the stream anymore.

But ideally, we should rework the handling of "airbyte" columns by ignoring them in order to properly chain normalization from raw tables